web/core/src/jsMain/kotlin/org/jetbrains/compose/web/elements/Base.kt

```diff
@@ -81,14 +79,25 @@ private class DomElementWrapper(override val node: Element): DomNodeWrapper(node
 
     fun updateAttrs(attrs: Map<String, String>) {
         node.getAttributeNames().forEach { name ->
-            if (name == "style") return@forEach
-            node.removeAttribute(name)
+            when (name) {
+                "style", "class" -> {
```

Comment:
What will happen if we have, say, uppercase STYLE or CLASS - for some reason? 
I don't remember that it's valid to be honest, just asking.